src/services/sentry.js

```diff
@@ -0,0 +1,12 @@
+import * as Sentry from '@sentry/browser';
+
+import config from '../config';
+
+Sentry.init({
+  // Set 'tracePropagationTargets' to control for which URLs distributed tracing should be enabled
+  dsn: config.sentryDsn,
+  tracePropagationTargets: [
+    'localhost',
+    /^https:\/\/(((test)|(www)).)?gogetta.nyc\/team/,
```

Comment:
One thing I missed is that technically these periods should probably be `\.`, but let's not be _that_ nitpicky. :)